jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/fileinput/JRobot.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.jdi.light.ui.bootstrap.elements.common.fileinput;
+
+import com.epam.jdi.tools.Timer;
+import java.awt.Robot;
+import java.awt.Toolkit;
+import java.awt.datatransfer.Clipboard;
+import java.awt.datatransfer.StringSelection;
+
+import static com.epam.jdi.light.common.Exceptions.safeException;
+import static com.epam.jdi.light.settings.WebSettings.logger;
+
+public final class JRobot {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'JRobot' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4515820)